target/linux/ar71xx/image/tp-link.mk

```diff
@@ -426,7 +426,7 @@ TARGET_DEVICES += tl-wpa8630
 define Device/tl-wr703n-v1
     $(Device/tplink-4mlzma)
     DEVICE_TITLE := TP-LINK TL-WR703N
-    DEVICE_PACKAGES := kmod-usb-core kmod-usb2
+    DEVICE_PACKAGES := kmod-usb-core kmod-usb-chipidea
     BOARDNAME := TL-WR703N
```

Comment:
The chipidea driver supports host mode as well. Agree usb2 is more sensible as a default since chipidea hasn't seen too much testing on this platform. And using it as a device requires (minor) hardware modification of the boards so no big deal if the default builds are not chipidea based.

Yes it's possible to load several device modules at the same time. Tried it a long time ago and it was working just fine. Here is some documentation on the topic - https://www.kernel.org/doc/Documentation/usb/gadget_multi.txt.

I've been recommended another board based on the same platform which is much more compact - [Zsun WiFi Card Reader](https://wiki.hackerspace.pl/projects:zsun-wifi-card-reader). Might suit you better.